web/assets/js/components/gamefen.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from "sevejs";
+import { gsap } from "gsap";
+
+const Gamefen = class extends Component {
```

Comment:
What is this used for?

Reply:
[sevejs](https://github.com/alexiscolin/seve-js) is a lib I coded few years ago to manage components lifecycle in creative static websites. So in our case, it manages the calls between components-classes public methods and init/destroy them at the right time. It also manage DOM manipulation.